psi4/src/psi4/ccresponse/optrot.cc

```diff
@@ -474,10 +487,21 @@ void optrot(std::shared_ptr<Molecule> molecule)
       outfile->Printf( "\n   Specific rotation using velocity-gauge electric-dipole Rosenfeld tensor.\n");
       outfile->Printf( "\t[alpha]_(%5.3f) = %10.5f deg/[dm (g/cm^3)]\n", params.omega[i], rotation_pl[i]);
 
-      if(params.wfn == "CC2")
-        Process::environment.globals["CC2 SPECIFIC ROTATION (VEL)"] = rotation_pl[i];
-      else if(params.wfn == "CCSD")
-        Process::environment.globals["CCSD SPECIFIC ROTATION (VEL)"] = rotation_pl[i];
+      /* grab omega in nm, rounded to nearest int */
+      long om_nm = std::lround((pc_c*pc_h*1e9)/(pc_hartree2J*params.omega[i]));
+
+      if(params.wfn == "CC2"){
+        /* build up a string */
+        std::stringstream tag;
+        tag << "CC2 SPECIFIC ROTATION (VEL) @ " << om_nm << "NM";
+        Process::environment.globals[tag.str()] = rotation_pl[i];
+      }
+      else if(params.wfn == "CCSD"){
+        /* build up a string */
+        std::stringstream tag;
+        tag << "CCSD SPECIFIC ROTATION (VEL) @ " << om_nm << "NM";
+        Process::environment.globals[tag.str()] = rotation_pl[i];
+      }
 
```

Comment:
Please add all of these entries to [the glossary](https://github.com/psi4/psi4/blob/master/doc/sphinxman/source/glossary_psivariables.rst), particularly including units and the fact that rounded to the nanometer.

Reply:
Done, let me know if I missed anything!